skin.json

```diff
@@ -199,6 +200,28 @@
         "enabled": true,
         "index": 4
       }
+    },
+    "controlBarButtons": {
```

Comment:
I think users might confuse these with the actual control bar buttons. Maybe we could change this to something like `controlBarSkipControls`?

Reply:
Added